util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/ReusableOrmSetupHolder.java

```diff
@@ -320,7 +321,7 @@ private void setupSessionFactory(Object testInstance) {
 				try {
 					clearAllData( sessionFactory );
 				}
-				catch (RuntimeException e) {
```

Comment:
I don't think this change is necessary? Why did you need to make `clearAllData` throw a `Throwable`?